### DISCUSSION TITLE
Feature: allow passing storyboard vtt via attribute

### DISCUSSION THREAD

**Discussion by rmarscher** [muxinc/elements#491] [Ideas]
Currently, the storyboard only seems to be returned if there is a playback ID.

We mostly use Mux Video, but have some external videos that have a storyboard.vtt in the same format as Mux.

https://github.com/muxinc/elements/blob/a2befb53200df08c40a79d7ed84e267cd1279648/packages/mux-player/src/index.ts#L698-L716

I think we could update the mux-player package to support passing in a storyboard url instead of only generating from the playback ID.

**Comment by dylanjha** [comment]
Hi @rmarscher,

Thanks for opening this. Mux player is intended to be used for only Mux sources. The main reason for that is that we can't really make guarantees that the playback engine (Hls.js) and the specific configurations we're using on Hls.js are optimal for any 'ol Hls source.

For that reason we didn't prioritize exposing a config for the storyboard URL, but we'll talk about it and see if we can add this anyway.
